recipes/stylist/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Please delete this file as well, the LICENSE file itself is included in the pypi tarball.